extension_templates/annotation.py

```diff
@@ -179,3 +182,28 @@ def _update(self, X, Y=None):
 
         # implement here
         # IMPORTANT: avoid side effects to X, fh
+
+    # todo: set default parameters, so that a test instance can be created
+    @classmethod
+    def get_test_params(cls):
+        """Get default parameters of the estimator.
+
+        Returns
+        -------
+        params : dict or list of dict, default = {}
+            Default parameters related to the estimator class
+        """
+        params = super().get_test_params()
```

Comment:
I would not implement any default logic here - leave it empty, this is not a base class but a "fill in" template.